### ISSUE TITLE
Change filled Braces and Wattle and Daub Braces to have consistent textures on opposing sides

### ISSUE THREAD

**Issue by Zetalight16** [MehVahdJukaar/Supplementaries#1138]
While the Timber Brace block has the braces mirrored across opposing sides, it changes to have a single bottom-right to top-left texture on all sides once filled. This makes filled brace blocks and wattle braces somewhat unsuitable for building decorative walls that are a single block thickness, as the bracing will look incorrect on the back side.

The behavior/appearance of these blocks should be changed such that a wall built with them looks the same from both sides.

**Comment by MehVahdJukaar** [comment]
So are you suggesting to have the textured mirrored for north west and down direction? Normally all textures face the same direction, so for example if you look at any square face block it's texture will always face the same way

**Comment by Zetalight16** [comment]
Yes--also, I am evidently mistaken about how empty Timber Braces are textured. I would apply this change to all forms of Brace.

A more concrete scenario: A player wants to have a 2x2 section of wall that appears to have a single continuous brace running diagonally through it, and uses two framed blocks and two braced blocks to accomplish this. Here's what that looks like from the north:
![image](https://github.com/user-attachments/assets/26f3b4f4-6d9b-4478-9346-956668d046c6)

vs from the south
![image](https://github.com/user-attachments/assets/374d0b17-2327-47a2-8280-b5fec894feca)

(snow backing layers added for contrast)

While the view from the north gives the desired effect, the view from the south does not. If, for example, a player wanted to use these blocks as the exterior wall of a half-timbered house, they would have to work around this discrepancy.

Note also that when looking _through_ a Timber Brace, the opposing face appears to be mirrored
(here matching the direction of the stair)
![image](https://github.com/user-attachments/assets/d21bdeb6-bae6-4136-8b79-4706f7aaca34)

But is not when viewed from the opposite side
(here, the same block opposing the direction of the same stair)
![image](https://github.com/user-attachments/assets/d6a226dd-c3ba-4e98-8e0c-b2a9f9d776c2)

So, as you said, I would think that Braced blocks should have direction-specific textures so that the direction of the brace appears consistent from opposite sides.
